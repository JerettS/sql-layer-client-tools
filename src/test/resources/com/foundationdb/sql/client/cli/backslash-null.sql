CREATE TABLE t ( label int, long_label int, a int);
INSERT INTO t (label, long_label, a) values (1,NULL,3), (4,NULL,6), (7,8,NULL);
SELECT * FROM t;

\null NULL

SELECT * FROM t;

\null

SELECT * FROM t;



