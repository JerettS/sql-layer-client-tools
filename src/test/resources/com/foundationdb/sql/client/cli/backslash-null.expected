ROWS: 0

ROWS: 3

 label | long_label | a 
-------+------------+---
     1 |            | 3 
     4 |            | 6 
     7 |          8 |   
(3 rows)

Format NULL as "NULL"
 label | long_label |  a   
-------+------------+------
     1 |       NULL |    3 
     4 |       NULL |    6 
     7 |          8 | NULL 
(3 rows)

Format NULL as ""
 label | long_label | a 
-------+------------+---
     1 |            | 3 
     4 |            | 6 
     7 |          8 |   
(3 rows)